app/javascript/components/ToastifyConfiguration.js

```diff
@@ -0,0 +1,10 @@
+export const ToastyConfiguration = {
+  position: "top-center",
+  type: "alert",
+  autoClose: 3000,
+  hideProgressBar: false,
+  closeOnClick: true,
+  pauseOnHover: true,
+  draggable: true,
+  progress: undefined,
+}
```

Comment:
Add a blank line here

Reply:
you Should install `the Blank Line at the End of File` extension in you Visual Code